packages/open-next/package.json

```diff
@@ -61,7 +61,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:opennextjs/opennextjs-aws.git",
+    "url": "git+https://github.com:opennextjs/opennextjs-aws.git",
```

Comment:
@dario-piotrowicz Wait is this correct actually ? 
The one in cloudflare is just this `"url": "https://github.com/opennextjs/opennextjs-cloudflare.git"`
Maybe that's why it doesn't work

Reply:
mh... `git+` should be correct as that what's included in the npm docs: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository
🤔 

I'm happy to give the http version a go too, I think that they're likely both valid

Reply:
@conico974 yeah the longer version is not helping, I do think we just need to wait for the next release 😕

Reply:
It was worth a try.
Ok i'll merge this then, we can revisit this if it still don't work on next release